docs/rooosss/pokemons.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/header.css">
+        <link rel="stylesheet" href="style/pokemon_page.css">
+        <link rel="stylesheet" href="style/filters.css">
+        <link rel="stylesheet" href="style/footer.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <section class="header__inner">
+                        <a class="header__main-link" href="index.html">
+                            <img src="./image/logo.svg" alt="Logo">
+                        </a>
+                        <div class="header__burger">
+                            <span class="header__burger-item"></span>
+                        </div>
+                        <nav class="navigation">
+                            <ul class="navigation__list">
+                                <li class="navigation__item">
+                                    <a class="navigation__link" href="calculator.html">Calculator</a>
+                                </li>
+                                <li class="navigation__item">
+                                    <a class="navigation__link" href="#">Pokemons</a>
+                                </li>
+                                <li class="navigation__item">
+                                    <a class="navigation__link" href="index.html">About</a>
+                                </li>
+                                <li class="navigation__item">
+                                    <a class="navigation__link" href="login.html">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+                    </section>
+                </div>
+            </header>
+            <main class="main">
+                <div class="container">
+                    <section class="filters">
```

Comment:
Same gaps now you have for filters. Proly you can lift up filters block out of main 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1ba3592b-0774-462f-baa7-34cfe03e41b3)